src/App.jsx

```diff
@@ -0,0 +1,16 @@
+import './App.css';
```

Comment:
the `App.css` file could be deleted if you won't add any styles for the component